tomcat8-nojsvc@.service

```diff
@@ -0,0 +1,36 @@
+# Systemd unit file for tomcat instances.
+#
+# To create clones of this service:
+# 0. systemctl enable tomcat8-nojsvc@name.service
+# 1. create catalina.base directory structure in
+#    /var/lib/tomcats/name
+# 2. set catalina base inside /etc/sysconfig/tomcat8@<name>
+
+[Unit]
+Description=Apache Tomcat 8 Web Application Container (No Jsvc)
+After=syslog.target network.target
+
+[Service]
+Type=forking
+
+Environment="NAME=%I"
+
+Environment=CATALINA_HOME=/opt/tomcat8
+Environment=JAVA_HOME=/usr/lib/jvm/jre-1.8.0
+Environment='CATALINA_OPTS=-Xms512M -Xmx1024M -server -XX:+UseParallelGC -Djava.awt.headless=true -Djava.security.egd=file:/dev/./urandom'
```

Comment:
Here we could use `EnvironmentFile`. An external environment file, like `/etc/sysconfig/mytomcat8` could be useful for manual changes that do not need `systemctl daemon-reload`.

Reply:
There is already defined a `EnvironmentFile` that can be used for specific application settings:
```
EnvironmentFile=-/etc/sysconfig/tomcat8@%I
```